### ISSUE TITLE
[Bug]: Ascending sort icon label incorrect

### ISSUE THREAD

**Issue by pcwarhurst** [pluralsight/design-system#2075]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

SortAscIcon has the wrong aria-label. It is currently 'sort desc icon'

### Expected behavior 🤔

The aria-label should be 'sort asc icon'.
![Screen Shot 2022-01-28 at 11 20 05 AM](https://user-images.githubusercontent.com/32023267/151600678-c26f2a99-6c2a-4a7d-904f-8bcba03b0553.png)

### Steps to reproduce 🕹

Steps:

1. Implement SortAscIcon
2. Inspect in dev-tools
3.
4.

### Context 🔦

I am trying to unit test in the coaching bc and need this label to check when it is in and not in the DOM
